clang/lib/CIR/CodeGen/CIRGenFunction.h

```diff
@@ -572,6 +575,11 @@ class CIRGenFunction : public CIRGenTypeCache {
 
   mlir::LogicalResult emitWhileStmt(const clang::WhileStmt &s);
 
+  /// Given an assignment `*lhs = rhs`, emit a test that checks if \p rhs is
+  /// nonnull, if 1\p LHS is marked _Nonnull.
+  void emitNullabilityCheck(LValue lhs, mlir::Value rhs,
```

Comment:
This is out of order alphabetically. A few other things are, so we need to do a cleanup pass anyway.